ThirdPartyAdapters/liftoffmonetize/liftoffmonetize/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -392,40 +390,25 @@ public View getBannerView() {
     return bannerLayout;
   }
 
-  public static BannerAdSize getVungleBannerAdSizeFromGoogleAdSize(Context context, AdSize adSize) {
-    ArrayList<AdSize> potentials = new ArrayList<>();
-    potentials.add(new AdSize(BannerAdSize.BANNER_SHORT.getWidth(),
-        BannerAdSize.BANNER_SHORT.getHeight()));
-    potentials.add(new AdSize(BannerAdSize.BANNER.getWidth(),
-        BannerAdSize.BANNER.getHeight()));
-    potentials.add(new AdSize(BannerAdSize.BANNER_LEADERBOARD.getWidth(),
-        BannerAdSize.BANNER_LEADERBOARD.getHeight()));
-    potentials.add(new AdSize(BannerAdSize.VUNGLE_MREC.getWidth(),
-        BannerAdSize.VUNGLE_MREC.getHeight()));
-
-    AdSize closestSize = MediationUtils.findClosestSize(context, adSize, potentials);
-    if (closestSize == null) {
-      return null;
-    }
-    Log.d(TAG,
-        "Found closest Liftoff Monetize banner ad size: " + closestSize + " for requested ad size: "
-            + adSize);
-
-    if (closestSize.getWidth() == BannerAdSize.BANNER_SHORT.getWidth()
-        && closestSize.getHeight() == BannerAdSize.BANNER_SHORT.getHeight()) {
-      return BannerAdSize.BANNER_SHORT;
-    } else if (closestSize.getWidth() == BannerAdSize.BANNER.getWidth()
-        && closestSize.getHeight() == BannerAdSize.BANNER.getHeight()) {
-      return BannerAdSize.BANNER;
-    } else if (closestSize.getWidth() == BannerAdSize.BANNER_LEADERBOARD.getWidth()
-        && closestSize.getHeight() == BannerAdSize.BANNER_LEADERBOARD.getHeight()) {
-      return BannerAdSize.BANNER_LEADERBOARD;
-    } else if (closestSize.getWidth() == BannerAdSize.VUNGLE_MREC.getWidth()
-        && closestSize.getHeight() == BannerAdSize.VUNGLE_MREC.getHeight()) {
-      return BannerAdSize.VUNGLE_MREC;
+  public static VungleAdSize getVungleBannerAdSizeFromGoogleAdSize(Context context, AdSize adSize) {
```

Comment:
@HuFangshuai @ManojBudumuru-vungle 
There is a little difference between iOS and Android to generate VungleAdSize.

For standard banners,

iOS:
We use `VungleAdSizeFromCGSize` since `VungleAdSizeBannerRegular` (and all other standard banner size) can not be used for in_feed placement ID.

Android:
We use `VungleAdSizeBannerRegular` (and any other standard banner sizes).

I discussed this part with @ManojBudumuru-vungle before, but we need to align both iOS and Android integration. 

https://github.com/Vungle/googleads-mobile-ios-mediation/blob/edfebe678485f39d1da39cf83979cc6adfeab079/adapters/LiftoffMonetize/LiftoffMonetizeAdapter/GADMAdapterVungleUtils.m#L28-L34

Reply:
Thanks let me know, I will update this pr to align with iOS.

Reply:
Thanks. I think we can use `VungleAdSizeFromCGSize` for standard banner sizes only for Google, since Google provides us banner size in CGSize instead of string. 

cc @ManojBudumuru-vungle

Reply:
Agree, pr updated.